db/migrate/20200818101637_add_missing_foreign_keys_to_html_variants_models.rb

```diff
@@ -0,0 +1,9 @@
+class AddMissingForeignKeysToHtmlVariantsModels < ActiveRecord::Migration[6.0]
+  def change
+    add_foreign_key :html_variant_successes, :html_variants, column: :html_variant_id, on_delete: :cascade, validate: false
+    add_foreign_key :html_variant_successes, :articles, column: :article_id, on_delete: :nullify, validate: false
```

Comment:
html variant success to variant itself is enforced in cascade mode
html variant success to article is enforced in nullify mode

same goes for trials below